src/lib/autocomplete/autocomplete.md

```diff
@@ -50,17 +49,17 @@ to the input's `mdAutocomplete` property.
 At this point, the autocomplete panel should be toggleable on focus and options should be
 selectable. But if we want our options to filter when we type, we need to add a custom filter.
 
-You can filter the options in any way you like based on the text input*. Here we will perform a
+You can filter the options in any way you wish based on the text input\*. Here we will perform a
```

Comment:
The asterisk here seems like a typo.

Reply:
I think it is intended as a footnote for the "optimal accessibility" section.

Reply:
Yeah, asterisk is intentional. Also: I actually think it sounds better with "like".

Reply:
Yeah I'll add it back. All I wanted to do was fix the broken link and ended up changing everything haha.